lib/trino-record-decoder/src/main/java/io/trino/decoder/DecoderColumnHandle.java

```diff
@@ -33,4 +36,9 @@
     String getMapping();
 
     String getDataFormat();
+
+    default Optional<Set<String>> getOneofs()
```

Comment:
nit: `getOneOfs` ?

Reply:
We've been styling as as `oneof`/`Oneof` to match how Protobuf styles it.

Reply:
Makes sense!